src/integration-tests/introspector/introspectTest.sh

```diff
@@ -39,6 +39,12 @@
 #     To check for ISTIO
 #         export ISTIO_ENABELD=true - this only test for Non Model in Image
 #
+#     To check for introspection of configured clusters with WebLogic 14.1.1.0 image.
+#     NOTE: Test for introspection of configured clusters only for Non Model in Image.
+#
+#         export WEBLOGIC_IMAGE_TAG=14.1.1.0
+#         introspectTest.sh
```

Comment:
Please update the "Usage:" and "Internal Test Flow" sections of the README file for this test to reflect this check-in's additions.  Also, if it's not too much trouble, please also update the README to discuss the ISTIO_ENABLED flag (I don't recall who added that - maybe Johnny).

Reply:
Looks good! Thanks!